userguide/sharing.rst

```diff
@@ -1130,12 +1130,10 @@ for more details.
    | netatalk             | Ease the co-existence of SMB and AFP shares.                                                                                    |
    |                      |                                                                                                                                 |
    +----------------------+---------------------------------------------------------------------------------------------------------------------------------+
-   | noacl                | Disables setting the ACL. If an extended ACL is present in the share connection path, all access to this share will be denied.  |
-   |                      |                                                                                                                                 |
-   |                      | When :guilabel:`Export Read Only` is set, all write bits are removed.                                                           |
-   |                      |                                                                                                                                 |
-   |                      | When :guilabel:`Export Read Only` is unset, write bits are added up to the mode defined by the SMB create and directory masks.  |
-   |                      | Remaining DOS modes are mapped to `chflags(1) <https://www.freebsd.org/cgi/man.cgi?query=chflags>`__ flags.                     |
+   | noacl                | Disable NT ACL support. If an extended ACL is present in the share connection path, all access to this share will be denied.    |
+   |                      | When the `Read-only attribute <https://www.oreilly.com/openbook/samba/book/ch05_03.html>`__ is set, all write bits are          |
+   |                      | removed. Disabling the *Read-only* attribute adds the write bits back to the share, up to *create mask* (*umask*).              |
+   |                      | Adding *noacl* requires adding the *zfsacl* object. *noacl* is incompatible with the *ixnas* VFS object.                        |
```

Comment:
"Adding *noacl* requires adding the *zfsacl* object." is still not clear to me.  Does "requires" mean it will add that for me automatically?  Does it mean that I can add noacl alone, it just won't work?  Or does it mean I have to add them both if I want noacl?

Reply:
This is a cherry-pick PR from an already approved and merged PR (https://github.com/freenas/freenas-docs/pull/1557). If you want further changes, then this PR and #1577 need to be closed until final text is created in #1557 .